src/app/features/tech-record/tech-record-routing.module.ts

```diff
@@ -19,6 +23,18 @@ const routes: Routes = [
     canActivateChild: [MsalGuard, RoleGuard],
     resolve: { load: TechRecordViewResolver }
   },
+  {
+    path: 'changeVehicleType',
```

Comment:
We use `kebab-case`, not `camelCase`, for our routes.

This should be `change-vehicle-type`.

Reply:
Nathan already caught this one too